dist/bin/scaladoc.bat

```diff
@@ -31,7 +31,7 @@ if defined JAVA_OPTS ( set _JAVA_OPTS=%JAVA_OPTS%
 set "_JAVACMD=!_JAVACMD:%%=%%%%!"
 
 call "%_JAVACMD%" %_JAVA_OPTS% %_JAVA_DEBUG% %_JAVA_ARGS% ^
--classpath "%_CLASS_PATH%" ^
+-classpath "@%_CLASS_PATH_FILE%" ^
```

Comment:
this should just be on its own, and instead put `-classpath` in the file also
```suggestion
"@%_CLASS_PATH_FILE%" ^
```

Reply:
see CliCommand.distill in here compiler/src/dotty/tools/dotc/config/CliCommand.scala